core/executor/atomic_match_executor.go

```diff
@@ -104,10 +104,10 @@ func (e *AtomicMatchExecutor) VerifyInputs(skipGasAmtChk bool) error {
 
 	// Check offer expired time.
 	if err := e.bc.VerifyExpiredAt(txInfo.BuyOffer.ExpiredAt); err != nil {
-		return errors.New("invalid BuyOffer.ExpiredAt")
+		return types.AppErrInvalidBuyOfferExpireTime
```

Comment:
line 102 needs update. Currently, for atomic match the trading asset should be one of the gas assets.

Reply:
OK, thanks!